.github/workflows/release.yml

```
@@ -0,0 +1,46 @@
+on:
+  push:
+    branches: [main]
+    paths-ignore:
+      - '**.md'
+      - 'docs/**'
+
+jobs:
+  release:
+    name: Release
+    runs-on: ubuntu-24.04
+    if: ${{ !contains(toLower(github.event.head_commit.message), 'skip ci') && !contains(toLower(github.event.head_commit.message), '[skip ci]') && !contains(toLower(github.event.head_commit.message), 'ci skip') && !contains(toLower(github.event.head_commit.message), '[ci skip]') && !contains(toLower(github.event.head_commit.message), 'skip-ci') }}
```

actionlint (1.7.10)

[error] undefined function "toLower". available functions are "always", "cancelled", "contains", "endswith", "failure", "format", "fromjson", "hashfiles", "join", "startswith", "success", "tojson"

(expression)